src/Entity/Framework/AwsStorage.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace App\Entity\Framework;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * Class AwsStorage
+ *
+ * @ORM\Entity(repositoryClass="App\Repository\Framework\AwsStorageRepository")
+ * @ORM\Table(name="aws_storage")
+ * @UniqueEntity("id")
+ */
+class AwsStorage
```

Comment:
This doesn't appear to be AWS specific.  We should probably try to have the storage as agnostic as possible when there is no reason not to be.